dotcom-rendering/src/experiments/tests/sign-in-gate-time-of-day.ts

```diff
@@ -0,0 +1,23 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const signInGateTimesOfDay: ABTest = {
+	id: 'SignInGateTimesOfDay',
+	start: '2020-05-20',
```

Comment:
To do: confirm start and end time

Reply:
Test needs to run between 4 days and 2.5 weeks. Choosing one month for a buffer and we can choose a switch off time later